src/pages/ResultsListPage.js

```diff
@@ -47,26 +45,22 @@ class ResultsListPage extends BaseResultsPage {
                             <div>Searching...</div>
                     }
                     secondaryText={
-                        this.state.meta ?
+                        this.state.statusCode == 402 ?
```

Comment:
Because the error and statusCode are set simultaneously I think this is ok as-is.